bin/mixserv_daemon.sh

```diff
@@ -119,13 +153,13 @@ case $1 in
     if [ -f $MIXSERV_PID_FILE ]; then
```

Comment:
better to use `yarn application -kill` for stopping yarn application.

https://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/YarnCommands.html#application